src/pcre2_auto_possess.c

```diff
@@ -1121,7 +1122,7 @@ for(;;)
   if (list[1] == 0) return TRUE;
   }
 
-  PCRE2_UNREACHABLE(); /* Control never reaches here */
+PCRE2_DEBUG_UNREACHABLE(); /* Control should never reach here */
 }
 
 
```

Comment:
the `PCRE2_UNREACHABLE()` below was fixed in a previous version of this patchset but regressed, a fix is forthcoming as [discussed](https://github.com/alexdowad/pcre2/commit/bb5cc085fd5d448e0e7b467fb25c958308fa883c#r147116369)